.gitignore

```diff
@@ -1,6 +1,8 @@
 
 app/views/issues/show.html.erb
 
+.idea
```

Comment:
This should be in your global environment, not in a project's gitignore. Not everyone uses RubyMine.